### PR TITLE
Keep unparsed user agent information in user_agent.original

### PR DESCRIPTION
**Pull request by kvch**
`user_agent.raw` has been renamed to `user_agent.original`. As this field is not yet released I am renaming it to follow conventions.

### CODE REVIEW

**Comment by ruflin** [comment]
Change LGTM. I think it will need a Changelog entry.

I have second thoughts if we should backport this to 6.x or not (not only raw to original, but the overall change).

**Comment by kvch** [comment]
Why? I think it can provide useful information is case of exotic user agents. I assume when someone investigates weird events happening in his/her network, it's possible that the person who might be lurking around leaves behind "unconventional" user agents.

**Comment by kvch** [comment]
Added changelog entry && rebased the branch

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -123,6 +123,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add tag "multiline" to "log.flags" if event consists of multiple lines. {pull}7997[7997]
 - Add haproxy module. {pull}8014[8014]
 - Release `docker` input as GA. {pull}8328[8328]
+- Rename user_agent.raw to user_ageint.original to follow ECS conventions. {pull}8537[8537]
```
Comment (jsoriano) [inline CHANGELOG.asciidoc]: If this hasn't been released yet, what about editing the previous entry instead of adding two for the same thing?

Comment (ruflin) [reply CHANGELOG.asciidoc]: ++

**Comment by ruflin** [comment]
@kvch Few thoughts around this that recently came up:

* Should we actually index `user_agent.original`? Is it mean to be searchable or more for reprocessing or looking into it when an event is found for the user (it will still be part of _source). This could save quite a bit of space on indexing
* The `user_agent` field will be a common field in 7.0 because of ECS. So the local fields will probably disappear. Should we add more fields now or wait for 7.0 until it's unified?

**Comment by kvch** [comment]
* +1 for not indexing it
* I would wait for 7.0. I have already renamed a few fields, because ECS is still a work in progress. I would rather minimize this overhead. If unavoidable, I would do it in a bulk before 7.0 is released.

**Comment by kvch** [comment]
@ruflin are you ok with merging this as is?

File: filebeat/module/apache2/access/_meta/fields.yml [hunk 2]
```diff
@@ -85,10 +85,11 @@
           type: keyword
           description: >
             The name of the operating system.
-        - name: raw
+        - name: original
           type: text
```
Comment (ruflin) [inline filebeat/module/apache2/access/_meta/fields.yml]: As we set `index: false` it should not matter here what type is defined. In ECS it seems we put `keyword`. Only reason I mention this is we should check later what shows up in the docs for non indexed fields.

**Comment by ruflin** [comment]
@kvch LGTM. Did an additional commit to resolve a CHANGELOG conflict.

File: filebeat/module/apache2/access/_meta/fields.yml (hunk 2 above)
Comment (kvch) [reply filebeat/module/apache2/access/_meta/fields.yml]: Got it. Thanks for the info.

**Comment by kvch** [comment]
Failing tests are unrelated.

**Pull request merged**